Better build log output with elapsed time

alpaix opened the pull request:
Added elapsed time to build output messages, replaced write-host with
custom Trace-Log function. Added common.ps1 file for misc helper methods.

alpaix committed:
Better build log output with elapsed time

Added elapsed time to build output messages, replaced write-host with
custom Trace-Log function. Added common.ps1 file for misc helper methods.

dnfclas commented:
Hi **@alpaix**, I'm your friendly neighborhood .NET Foundation Pull Request Bot (You can call me DNFBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by .NET Foundation and real humans are currently evaluating your PR.</p>

TTYL, DNFBOT;

alpaix committed:
Packing multiple Core projects in single dnu pack

Calling dnu pack [p1, p2, ..., pn] for all projects packing.
Added -Fast switch to run less reliable parallel restore in local builds
only.

emgarten commented:
:shipit:

emgarten commented:
:rocket:
